### PR TITLE
add metatype description

### PR DESCRIPTION
**Pull request by utle**
Add metatype description for management authorization user/group accessId

### CODE REVIEW

**Comment by utle** [comment]
#build

File: dev/com.ibm.ws.management.security/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -21,5 +21,13 @@ role.administrator.desc=A collection of users and/or groups assigned the server
 role.user=User name
 role.user.desc=User assigned a role.
 
+role.userAccessId=User access ID
+role.userAccessId.desc=A user access ID in the general form user:realmName/userUniqueId. A value will be generated if one is not specified.
```
Comment (sishida) [inline dev/com.ibm.ws.management.security/resources/OSGI-INF/l10n/metatype.properties]: We try to keep sentences in present tense. How about "A value **is** generated if one is not specified."?

Comment (utle) [reply dev/com.ibm.ws.management.security/resources/OSGI-INF/l10n/metatype.properties]: fixed

**Comment by utle** [comment]
#build

**Comment by sishida** [comment]
Thank you for making the changes! I have added my "ID reviewed" label.

**Comment by utle** [comment]
#build

**Comment by utle** [comment]
#build

**Comment by utle** [comment]
There are some failures but I don't think adding the metatype description cause these failures

**Comment by utle** [comment]
Fixed #3626

**Pull request merged**